syntax = "proto3";

package suggestions;

service SuggestionService {
    rpc GetSuggestions (GetSuggestionsRequest) returns (GetSuggestionsResponse);
}

message GetSuggestionsRequest {
    string title = 1;
}

message GetSuggestionsResponse {
    repeated Book suggestedBooks = 1;
}

message Book {
    uint32 book_id = 1;
    string title = 2;
    string author = 3;
}




